omnibus/config/software/datadog-agent.rb

```diff
@@ -131,23 +131,30 @@
     end
   end
 
+  # Process agent
   if windows?
     platform = windows_arch_i386? ? "x86" : "x64"
     # Build the process-agent with the correct go version for windows
     command "invoke -e process-agent.build --python-runtimes #{py_runtimes_arg} --major-version #{major_version_arg} --arch #{platform} --flavor #{flavor_arg}", :env => env
 
     copy 'bin/process-agent/process-agent.exe', "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent"
+  else
+    command "invoke -e process-agent.build --python-runtimes #{py_runtimes_arg} --major-version #{major_version_arg} --flavor #{flavor_arg}", :env => env
+    copy 'bin/process-agent/process-agent', "#{install_dir}/embedded/bin"
+  end
 
+  # System-probe
+  if windows?
     unless windows_arch_i386?
       if ENV['WINDOWS_DDNPM_DRIVER'] and not ENV['WINDOWS_DDNPM_DRIVER'].empty?
         ## don't bother with system probe build on x86.
         command "invoke -e system-probe.build --windows"
         copy 'bin/system-probe/system-probe.exe', "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent"
       end
     end
-  else
-    command "invoke -e process-agent.build --python-runtimes #{py_runtimes_arg} --major-version #{major_version_arg} --flavor #{flavor_arg}", :env => env
-    copy 'bin/process-agent/process-agent', "#{install_dir}/embedded/bin"
+  elsif linux?
+    command "invoke -e system-probe.build-sysprobe-binary"
+    copy "bin/system-probe/system-probe", "#{install_dir}/embedded/bin"
```

Comment:
Shouldn't the binary be generated directly into install_dir like other binaries are?

Reply:
All the binaries except the core agent are put into `#{install_dir}/embedded/bin`